OMCompiler/SimulationRuntime/c/util/boolean_array.c

```diff
@@ -922,3 +922,33 @@ void fill_alloc_boolean_array(boolean_array_t* dest, modelica_boolean value, int
         boolean_set(dest, i, value);
     }
 }
+
+modelica_boolean min_boolean_array(const boolean_array_t a)
```

Comment:
Could you add a small description to the functions what it does? In a month a need to read the whole function to get what it's doing.
E.g.: Returns minimum of given boolean array a.

Reply:
Done